lib/DoctrineCodingStandard/Sniffs/Classes/ExceptionInterfaceNamingSniff.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineCodingStandard\Sniffs\Classes;
+
+use DoctrineCodingStandard\Helpers\UseStatementHelper;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use const T_EXTENDS;
+use const T_INTERFACE;
+use const T_OPEN_CURLY_BRACKET;
+use function array_combine;
+use function array_map;
+use function count;
+use function explode;
+use function preg_grep;
+use function preg_match;
+use function trim;
+
+final class ExceptionInterfaceNamingSniff implements Sniff
+{
+    private const CODE_NOT_AN_EXCEPTION = 'NotAnException';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function register() : array
+    {
+        return [T_INTERFACE];
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
Document `@param int $pointer` instead.

Reply:
Code sniffer doesn't like having that param annotation and wants me to use the int type hint in the method.